src/OrbitBase/TypedefTest.cpp

```diff
@@ -368,6 +370,19 @@ TEST(Typedef, WrapperWithArithmeticsHasPlusAndConvertsArgument) {
   EXPECT_EQ(*(Add(a, b)), kNanos + kMicros);
 }
 
+TEST(Typedef, PostIncrement) {
+  WrapperWithArithmetics<int> a(kAValue);
+  WrapperWithArithmetics<int> old = a++;
+  EXPECT_EQ(*a, kAValue + 1);
+  EXPECT_EQ(*old, kAValue);
+}
+
+TEST(Typedef, PreIncrement) {
```

Comment:
Why not adding a similar test than above?

```
  WrapperWithArithmetics<int> old = ++a;
  EXPECT_EQ(*a, kAValue + 1);
  EXPECT_EQ(*old, kAValue + 1);
```

Reply:
I don't see what it accomplishes on top of the test we already have. Your version doesn't test that the operator returns a reference to `*this`.